.github/workflows/tests.yml

```diff
@@ -33,3 +33,41 @@ jobs:
       - name: Test with pytest
         run: |
           python -m pytest tests/
+
+  build_windows:
+    needs: pre_job
+    if: needs.pre_job.outputs.should_skip != 'true'
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: ["3.10","3.12"]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: garak
+
+      - name: Checkout ecoji for modified windows install
+        uses: actions/checkout@v3
+        with:
+          repository: mecforlove/ecoji-py
+          path: ecoji-py
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          cd ecoji-py
+          echo "mitigate" > README.md
+          pip install setuptools
+          python setup.py install
```

Comment:
Since this is a `mitigation` and the project in involved has not had any changes since before the deprecation this is just temporary artifact of the workaround and acceptable in this context.